homework.md

```diff
@@ -44,70 +44,73 @@ Using CSS, how would I select every `<h1>` tag on the page?
 
 What is JavaScript?
 
-A programming language used on the internet
+Javascript is a scripting language that enables you to create dynamically updating content, control multimedia, animate images. It's a dynamic programming language that's used for web development, in web applications, for game development, and lots more.
 
 ## Question 8
 
 What three problem does Git & Github solve?
 
-When people want to show off code to each other they can put it on Github
+GitHub is a website for developers and programmers to collaboratively work on code.
+
+The primary benefit of GitHub is its version control system, which allows for seamless collaboration without compromising the integrity of the original project. 
```

Comment:
Note that Git is the version control system, and that GitHub is that platform that hosts the usage of Git for developers

Reply:
> Note that Git is the version control system, and that GitHub is that platform that hosts the usage of Git for developers

Yes. I've learned more about Git since we've been using it each week. This was my first foray into figuring out what all the terminology meant. Thank you for helping me to not trip up over the two definitions